controllers/admin/AdminSearchController.php

```diff
@@ -180,7 +180,16 @@ public function postProcess()
             /* Invoices */
             if ($searchType == 4) {
                 if (Validate::isOrderInvoiceNumber($this->query) && ($invoice = OrderInvoice::getInvoiceByNumber($this->query))) {
```

Comment:
I think we can remove this part. 

`Validate::isOrderInvoiceNumber($this->query) && `

When i test the PR it's work with `#FA000001 ` but not with `1` or i can open another ici for that ?

Reply:
Yes